qpup_hw/include/qpup_hw/qpup_hw.hpp

```diff
@@ -57,9 +57,9 @@ class QPUPHW : public hardware_interface::RobotHW {
     float iq_measured = 0;
     float vbus_voltage = 0;
 
-    uint16_t axis_state_cmd = 0;
-    uint8_t control_mode_cmd = 0;
-    uint16_t input_mode_cmd = 0;
+    uint16_t axis_state_cmd = 1;
```

Comment:
`odrive_state_msgs::SetAxisState::Request::Type::AXIS_STATE_IDLE`? Shouldn't really matter though since you `init()` before anything else